server/routes/routes.js

```diff
@@ -0,0 +1,2 @@
+//all auction routes here
+
```

Comment:
Let's also make a route here then link it with the dummy controller.

Reply:
Alright